<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="My_Resources.qrc">
    <normaloff>:/icons/cloud.ico</normaloff>:/icons/cloud.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Download File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Delete File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Upload File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="undoButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Undo Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reloadButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Reload Storage</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="adminPanelButton">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Admin Panel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="logoutButton">
            <property name="text">
             <string>Log Out</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mainLayout" stretch="2,0,3,3,1">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QTreeView" name="fileViewer">
            <property name="styleSheet">
             <string notr="true">font-size: 20px;
background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="imagePreview">
            <property name="styleSheet">
             <string notr="true">width:200px;
height:100px;
background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="responseDisplay">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateTextFile">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0" columnminimumwidth="0,1,1,0">
        <item row="11" column="3">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>285</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>285</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="2">
         <widget class="QFrame" name="loginFrame">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(61, 43, 68);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0">
           <item row="5" column="1">
            <widget class="QLineEdit" name="passwordInput">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(61, 56, 70)</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QPushButton" name="registerAccount">
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Register</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="usernameInput">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(61, 56, 70)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="loginLabel">
             <property name="styleSheet">
              <string notr="true">font: 200 11pt &quot;Cascadia Code&quot;;</string>
             </property>
             <property name="text">
              <string>Log In Panel</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="submitLogin">
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Log in</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QFrame" name="frame_2">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(61, 43, 68);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="1">
            <widget class="QPushButton" name="validateButton">
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Validate Port</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="stateLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(36, 31, 49);
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
color:rgb(165, 29, 45);</string>
             </property>
             <property name="text">
              <string>Disconnected</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="portInput">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(61, 56, 70)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_13">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_14">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adminPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="adminLayout" rowstretch="0,0" columnstretch="0,0,0,0" columnminimumwidth="0,0,0,0">
          <item row="1" column="1">
           <widget class="QTableWidget" name="logsTable">
            <property name="styleSheet">
             <string notr="true">font-size: 12px;
background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Action</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Hour</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Day</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>File</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>User</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QTreeView" name="globalStorage">
            <property name="styleSheet">
             <string notr="true">font-size: 20px;
background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="mainPanelButton">
            <property name="text">
             <string>Main Panel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTableWidget" name="userCredentialsTable">
            <property name="styleSheet">
             <string notr="true">font-size: 12px;
background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Username</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Password</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Admin Privileges</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="globalImageDisplay">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QTextEdit" name="globalTextEditor">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(36, 31, 49);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="registerPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="registerLayout">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(61, 43, 68);
border-radius: 5px;
border-style:outset;
border-width:2px;
border-color:rgb(34, 22, 37);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,0">
             <item row="6" column="1">
              <widget class="QLineEdit" name="regPassword">
               <property name="styleSheet">
                <string notr="true">background-color:rgb(61, 56, 70)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="regUsername">
               <property name="styleSheet">
                <string notr="true">background-color:rgb(61, 56, 70)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="registerLabel">
               <property name="styleSheet">
                <string notr="true">font: 200 11pt &quot;Cascadia Code&quot;;</string>
               </property>
               <property name="text">
                <string> Register Panel</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QPushButton" name="loginReturn">
               <property name="cursor">
                <cursorShape>OpenHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Back to Login</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QPushButton" name="registerButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="cursor">
                <cursorShape>OpenHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Register</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="Line" name="line_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="5" column="0">
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="My_Resources.qrc"/>
 </resources>
 <connections/>
</ui>
